modules/capi-yaml-infrastructure-aws.adoc

```diff
@@ -11,12 +11,12 @@ The infrastructure resource is provider-specific and defines properties that are
 [source,yaml]
 ----
 apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
-kind: AWSCluster <1>
+kind: AWSCluster # <1>
 metadata:
-  name: <cluster_name> <2>
+  name: <cluster_name> # <2>
   namespace: openshift-cluster-api
 spec:
```

Comment:
Seems this part added new parameters, spec looks like below @huali9 please help to confirm
```
spec:
  controlPlaneEndpoint:
    host: api-int.zhsun-aws29.qe.devcluster.openshift.com
    port: 6443
  region: us-east-2
```

Reply:
I made this guess for that part:

```
spec:
  controlPlaneEndpoint: # <3>
    host: <control_plane_endpoint_address>
    port: 6443
  region: <region> # <4>
----

<3> Specify the address of the control plane endpoint and the port to use to access it.
<4> Specify the AWS region.
```